src/main/java/seedu/duke/TestManager.java

```diff
@@ -0,0 +1,37 @@
+package seedu.duke;
+
+import java.util.ArrayList;
+
+public class TestManager {
+    static ArrayList<Event> test = new ArrayList<>();
```

Comment:
Making the ArrayList static seems unnecessary. Maybe consider `private ArrayList<Event> tests = new ArrayList<>();`